src/main/java/org/broadinstitute/hellbender/tools/picard/analysis/CollectInsertSizeMetrics.java

```diff
@@ -51,7 +51,7 @@
     public float MINIMUM_PCT = 0.05f;
 
     @Argument(shortName="LEVEL", doc="The level(s) at which to accumulate metrics.  ")
-    private Set<MetricAccumulationLevel> METRIC_ACCUMULATION_LEVEL = CollectionUtil.makeSet(MetricAccumulationLevel.ALL_READS);
+    public Set<MetricAccumulationLevel> METRIC_ACCUMULATION_LEVEL = CollectionUtil.makeSet(MetricAccumulationLevel.ALL_READS);
```

Comment:
While your at it, it might be worth replacing this call to `CollectionUtils.makeSet()` with `EnumSet.of()`